<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RMP52SJ" failures="2" tests="5" name="Test" time="0.095" errors="0" timestamp="2022-05-20T23:43:58 BDT">
  <testcase classname="tlistaner.TestngLDemo" name="test5" time="0.012">
    <skipped/>
  </testcase> <!-- test5 -->
  <testcase classname="tlistaner.TestngLDemo" name="test4" time="0.009">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at tlistaner.TestngLDemo.test4(TestngLDemo.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="tlistaner.TestngLDemo" name="test3" time="0.003">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at tlistaner.TestngLDemo.test3(TestngLDemo.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="tlistaner.TestngLDemo" name="test2" time="0.001"/>
  <testcase classname="tlistaner.TestngLDemo" name="test1" time="0.002"/>
</testsuite> <!-- Test -->
